### ISSUE TITLE
Sandbox build workaround for `no such table: main.catalogue_product__old`

### ISSUE THREAD

**Issue by MichaelSaah** [django-oscar/django-oscar#2901]
This ticket extends #2898 and specifically addresses users attempting to setup the oscar sandbox. I believe it would be best to note the issue described here in the README/docs until a patched version of Django is released to PyPI.

When attempting to `make` the django-oscar sandbox with the latest sqlite and Django versions (3.26.0 and 2.1.4 for me, respectively), one will likely run into the following exception:

```
django.db.utils.OperationalError: Problem installing fixture '/Users/Mike/website/django-oscar/sandbox/fixtures/child_products.json': Could not load catalogue.ProductCategory(pk=1): no such table: main.catalogue_product__old
```

This is due to the bug noted in #2898 and addressed in this Django PR: https://github.com/django/django/pull/10733

A simple temporary workaround is to use an older sqlite. I switched to 3.19.0 and the build succeeds. The process for doing this is OS/environment specific.

**Comment by officerebel** [comment]
So For the bug in the sandbox of oscar one should use an older sql lite database?3.19.0 to prevent saving products in the shop and get did of this error? 
Where to get it? https://www.sqlite.org/src/timeline

Request Method: | POST
-- | --
http://localhost:8000/nl/dashboard/catalogue/products/create/koffiebbonen/
2.1.4
OperationalError
no such table: main.catalogue_product__old
/Users/timvogt/.virtualenvs/django-oscar-jB0wsp9N/lib/python3.7/site-packages/django/db/backends/sqlite3/base.py in execute, line 296
/Users/timvogt/.virtualenvs/django-oscar-jB0wsp9N/bin/python
3.7.1
['/Users/timvogt/Software_projects/Backend/django-oscar/sandbox',  '/Users/timvogt/.virtualenvs/django-oscar-jB0wsp9N/lib/python37.zip',  '/Users/timvogt/.virtualenvs/django-oscar-jB0wsp9N/lib/python3.7',  '/Users/timvogt/.virtualenvs/django-oscar-jB0wsp9N/lib/python3.7/lib-dynload',  '/usr/local/Cellar/python/3.7.1/Frameworks/Python.framework/Versions/3.7/lib/python3.7',  '/Users/timvogt/.virtualenvs/django-oscar-jB0wsp9N/lib/python3.7/site-packages',  '/Users/timvogt/Software_projects/Backend/django-oscar/src']
zo, 23 Dec 2018 08:15:17 +0000

**Comment by Penagwin** [comment]
Heads up, django/django#10733 was merged into Django 2.1.5 so if you upgrade to that it should be fixed.

**Comment by solarissmoke** [comment]
Thanks for the heads up. Closing as this is fixed upstream.
